src/components/progress/progress.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Input, ViewEncapsulation, HostBinding } from '@angular/core';
+
+@Component({
+  selector: 'ion-progress',
+  templateUrl : './progress.html',
+  encapsulation: ViewEncapsulation.None
+})
+export class Progress {
+
+  constructor( ) { }
+
+  /**
+   * @input {'determinate' | 'indeterminate' | 'buffer' | 'query'} The current mode of the progress indicator
+   */
+  @Input()
+  @HostBinding('class')
```

Comment:
Hmm yeah can we use `host` please just to keep it consistent? Also with v4 we will most likely end up using `host` anyways so it will be easier to port the component. 🙂